.github/workflows/publish-to-package-repositories.yaml

```
@@ -56,72 +55,73 @@ jobs:
+          echo "WINDOWS_SHA=${WINDOWS_SHA}" >> "${GITHUB_ENV}"
 
           version="$(cat /tmp/component-descriptor.yaml | \
             yq '.component.version'
           )"
-          echo "version=${version}" >> "${GITHUB_OUTPUT}"
+          echo "version=${version}" >> "${GITHUB_ENV}"
 
       - name: Use dummy hashes (skip‑build)
-        id: dummy-hashes
-        if: ${{ inputs.component-descriptor == '' }}
+        if: github.event_name == 'workflow_dispatch' && inputs.skip_build == true
```

actionlint (1.7.12)

[error] property "skip_build" is not defined in object type {component-descriptor: string; component-descriptor-artifact: string; dry_run: bool; version: string}

(expression)